manifest/org.zeith/VDFaceTracking/info.json

```diff
@@ -5,6 +5,13 @@
 	"category": "Hardware Integration",
 	"sourceLocation": "https://github.com/Zeitheron/VDFaceTracking",
 	"versions": {
+		"1.0.3": {
```

Comment:
Version string "1.0.2" does not match version being added

Reply:
O hecc!